plugins/CoreHome/angularjs/report-export/reportexport.directive.js

```diff
@@ -253,7 +256,7 @@
                     if (scope.maxFilterLimit > 0) {
                         scope.$watch('reportLimit', function (newVal, oldVal) {
                             if (parseInt(newVal, 10) > parseInt(scope.maxFilterLimit, 10)) {
-                                scope.reportLimit = oldVal;
+                                scope.limitContainer.reportLimit = oldVal;
```

Comment:
Using an object was the only way I could get this to work. This bug (described in the issue description) is also in 4.x-dev.